lib/assets/schemas/collections/umm-cmn-json-schema.json

```diff
@@ -343,7 +343,7 @@
         "properties": {
             "Format": {
                 "description": "The format of the data.",
-                "$ref": "#/definitions/GetDataTypeFormatEnum"
+                "$ref": "#/definitions/GetServiceTypeFormatEnum"
```

Comment:
I'm wondering why this is showing up now, did we miss it before?

Reply:
I guess so